oaknational/Oak-Web-Application
# Feat/specialist programme page - LESQ-499

## Description

Decided not to make a `view` for this page. There is a lot of logic around exam boards and tiers and is not needed for the specialist programme page and adding specialist logic to existing components with be messy. Instead I have refactored some of the SubjectProgrammeListing components so they can be reused when we make the speciallist programme page and I have created a SpecialistHeaderListing component which is in storybook.

The curriculum button I have commented out for now. and created this ticket.
https://www.notion.so/oaknationalacademy/I-want-to-be-able-to-download-specialist-curriculum-maps-e3a1c4e425384349ba7a8984494639d9

Music year: {owa_music_year}

- Adds SpecialistHeaderListing
- Adds SpecialistProgrammeListing
- Refactors SubjectProgrammeListing components so they can be used on Specialist page
- Refactors tracking for programming to page level

## Issue(s)

Fixes #

## How to test

SpecialistHeaderListing in storybook

1. Go to https://deploy-preview-2200--oak-web-application.netlify.thenational.academy
Programme pages should behave the same.
Existing avo events should still be sent.



## Checklist
